packages/spec/README.md

```diff
@@ -36,12 +24,20 @@ export interface StandardSchema<Input = unknown, Output = Input> {
    */
   readonly "~validate": StandardValidate<Output>;
   /**
-   * The stored type information of the schema.
+   * Inferred types associated with the schema.
    */
-  readonly "~types"?: StandardTypes<Input, Output> | undefined;
+  readonly "~types": StandardTypes<Input, Output> | undefined;
 }
 ```
 
+- `~standard` stores the version number and can be used to test whether an object is a Standard Schema. 
+
+- `~vendor` stores the name of the schema libarry. This can be useful for performing vendor-specific operations in special cases. 
+
+- `~validate` is a function that validates unknown input and returns the output of the schema if the input is valid or an array of issues otherwise. This can be discriminated by checking whether the `issues` property is `undefined`.
+
+- `~types` is used to associate type metadata with the schema. This property should be declared on the schema's type, but is not required to exist at runtime. Authors implementing the schema are encouraged to use TypeScript's `declare` keyword or other means of avoiding runtime overhead. `unknown` can be used as a fallback for `input` and/or `output` depending what inference the implementation supports.
```

Comment:
If only the input type is defined, the current implementation intentionally defaults the output to the input. There is no need to use `unknown` in my opinion. Or is there a use case?

Can you give a code example using the `declare` keyword to avoid runtime overhead?

Reply:
I will change this back, I think I just added it when I had made the property required to address concerns related to library authors who don't have one or both sides inferred.